packages/shared/src/components/fields/SearchField.tsx

```diff
@@ -73,17 +87,36 @@ export const SearchField = forwardRef(function SearchField(
     setInput(null);
   };
 
+  const isPrimary = fieldType === 'primary';
+  const isSecondary = fieldType === 'secondary';
+  const SearchIcon = focused ? MagnifyingFilledIcon : MagnifyingOutlineIcon;
```

Comment:
You don't have to do this now, but could you note that we add a ticket to make a hook for this?

Reply:
That's a great idea.